test/default-xrp-client-test.ts

```diff
@@ -461,4 +468,135 @@ describe('Default Xpring Client', function(): void {
       done()
     })
   })
+
+  it('Payment History - successful response', async function(): Promise<void> {
+    // GIVEN a DefaultXRPClient.
+    const xrpClient = new DefaultXRPClient(fakeSucceedingNetworkClient)
+
+    // WHEN the payment history for an address is requested.
+    const paymentHistory = await xrpClient.paymentHistory(testAddress)
+
+    const expectedPaymentHistory: Array<XRPTransaction> = []
+    for (const getTransactionResponse of testGetAccountTransactionHistoryResponse.getTransactionsList()) {
```

Comment:
If this is the same loop as above, it could be useful to refactor it into a helper function that can be reused.

Reply:
yeah... related to the comments above between me and keefer.  This lends enough impetus so I just went ahead and refactored it and created a new XRPTestUtils class.  This'll help inspire us to keep things well maintained as the tests continue to expand.

Reply:
Got it!